src/core/TriangleSplitter.js

```diff
@@ -110,6 +120,23 @@ export class TriangleSplitter {
 		const { normal, triangles } = this;
 		triangle.getNormal( _triangleNormal ).normalize();
 
+		// compute triangleMinEdgeSize:
+		let triangleMinEdgeSizeSq = Infinity;
+		const arr = [ triangle.a, triangle.b, triangle.c ];
```

Comment:
This array with the same layout should be available as `triangle.points` since the triangle argument is of type `ExtendedTriangle` (if I recall correctly). If we use the existing array we can cut down on some small allocation since this function gets called a lot.

Reply:
Good catch

Reply:
Unfortunately triangle.points is undefined

Reply:
Gotcha, thanks! Looks like a Triangle is being passed in, then.